main.go

```diff
@@ -3,13 +3,38 @@ package main
 import (
 	"log"
 	"net/http"
+	"net/http/httputil"
+	"net/url"
 	"observability-remote-write-proxy/pkg/remotewrite"
 )
 
+const (
+	prefixHeader = "X-Forwarded-Prefix"
+)
+
 func main() {
-	handler := http.HandlerFunc(handleRequestSuccess)
-	http.Handle("/", handler)
-	err := http.ListenAndServe(":8080", nil)
+	upstreamUrl, err := url.Parse("http://localhost:9091/api/v1/write")
```

Comment:
let's remember to make the upstream url and the needed upstream url credentials configurable

Reply:
thanks @machi1990 , just changed those!